Tests/UnitTests/Core/Material/BinnedSurfaceMaterialAccumulaterTests.cpp

```diff
@@ -40,11 +37,11 @@ BOOST_AUTO_TEST_CASE(InvalidSetupTest) {
   };
 
   for (auto [is, surface] : enumerate(surfaces)) {
-    surface->assignGeometryId(GeometryIdentifier().withSensitive(is + 1));
+    surface->assignGeometryId(GeometryIdentifier().setSensitive(is + 1));
```

Comment:
Im planning to deprecate the `setSensitive` and related setters.

Reply:
oh right sorry I blindly copied that over from an older branch